mappings/net/minecraft/client/render/entity/EntityRenderDispatcher.mapping

```diff
@@ -15,45 +15,55 @@ CLASS net/minecraft/class_898 net/minecraft/client/render/entity/EntityRenderDis
 	METHOD <init> (Lnet/minecraft/class_1060;Lnet/minecraft/class_918;Lnet/minecraft/class_3296;Lnet/minecraft/class_327;Lnet/minecraft/class_315;)V
 		ARG 1 textureManager
 	METHOD method_17145 register (Lnet/minecraft/class_1299;Lnet/minecraft/class_897;)V
-	METHOD method_23161 fireVertex (Lnet/minecraft/class_4587$class_4665;Lnet/minecraft/class_4588;FFFFF)V
-		ARG 0 matrix
-		ARG 1 vertexConsumer
+	METHOD method_23161 drawFireVertex (Lnet/minecraft/class_4587$class_4665;Lnet/minecraft/class_4588;FFFFF)V
+		ARG 0 entry
+		ARG 1 vertices
```

Comment:
`vertexConsumer`.

Reply:
If it's a VertexConsumer naming it `vertices` is intended.